apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md

```diff
@@ -6,73 +6,65 @@ hasSubPageNav: true
 
 If the customer is using XP and moving to an XM + Personalize, their existing deployment may change slightly. With XP, they needed xConnect and supporting pieces to support this environment, but with XM + Personalize, there is no longer a need to have a dependency on this additional infrastructure. However, any other changes to the Platform as a Service approach would not require additional modifications.
 
 If a customer is currently using XM On-Prem or Managed Cloud and moving to XM Cloud + Embedded Personalize (or standalone Personalize) there would be a change in their Dev Ops process, due to the nature of using XM Cloud which is a Headless SaaS product. Some technologies are no longer supported by the XM Cloud infrastructure, and therefore would require a change in strategy for the organization.
 
-**What can we personalize? Do we use the same rule editor that we do in XP now?\*\*** \*\*
+### What can we personalize? Do we use the same rule editor that we do in XP now?
 
-This is a two-part question, regarding the first question.Using the headless personalization style that comes with Sitecore Personalize, you can build almost any personalization scenario that suits your requirements.
+This is a two-part question, regarding the first question. Using the headless personalization style that comes with Sitecore Personalize, you can build almost any personalization scenario that suits your requirements.
 
 If the data exists in the system, then it can be used to personalize.
 
-Second question: I assume this is related to the XP to XM + Personalize topic, which in that case, no, there is a different editor in the Personalize product.
+Second question: No, there is a different editor in the Personalize product.
 
-**Does Personalize work at the component level to swap out content or just pop-ups? **
+### Does Personalize work at the component level to swap out content or just pop-ups?
 
 Personalization is not restricted to just pop-up functionality but instead can apply to page level or component level variations. Offers stored in Sitecore Personalize paired with Web Experiences can allow marketers to customize various parts of a webpage, from within Sitecore Personalize using Html/CSS/JavaScript. In addition, using a Full Stack headless experience, any content irrespective of the source (including XM or Content Hub) can be personalized.
 
-**Will my potential Personalize implementation still be compatible and work if I later move into XM SaaS?**
+### Will my potential Personalize implementation still be compatible and work if I later move into XM Cloud?
 
 If the user has purchased Standalone Personalize (or CDP, SmartHub CDP) then they will keep that instance if they move to XM Cloud. The embedded Personalization in Pages will be enhanced with additional condition templates and the user will be able to update the experiences created in Pages by going into the Sitecore Personalize app.
 

```

Comment:
Weird, I don't really see this in the code, but there is a ton of odd white space underneath a few of the questions on this page.  So I'm not sure if there is much that could be done, but something to look into.

Reply:
@dylanyoung-dev : Can you provide more details on this? I can see some headings that are the wrong level (h2 vs h3) but I'm not sure if I'm addressing the right issues on this page.

Reply:
Marking this as resolved as the headings change seems to make it look as expected.